README.md

```diff
@@ -0,0 +1,56 @@
+Instrumental .NET Agent
+================
+
+Instrumental is a [application monitoring platform](https://instrumentalapp.com) built for developers who want a better understanding of their production software. Powerful tools, like the [Instrumental Query Language](https://instrumentalapp.com/docs/query-language), combined with an exploration-focused interface allow you to get real answers to complex questions, in real-time.
+
+This agent supports custom metric monitoring for .NET applications.
+
+Getting Started
+===============
+
+If you are using NuGet, add Instrumental.NET to your packages.config:
+
+```xml
+<id="Instrumental" version="0.2.0" targetFramework="net45" />
+```
+
+Or, download [Instrumental.dll](https://github.com/Instrumental/Instrumental.NET/releases/latest).
```

Comment:
I think this link will work because github is cool about redirecting, but probably wouldn't hurt to update the repo name here.